packages/backend-common/api-report.md

```diff
@@ -335,17 +353,18 @@ export const legacyPlugin: (
           logger: LoggerService;
           permissions: PermissionsService;
           scheduler: SchedulerService;
-          tokenManager: TokenManagerService;
           reader: UrlReaderService;
-          identity: IdentityService;
         },
         {
           logger: (log: LoggerService) => Logger;
           cache: (cache: CacheService) => {
             getClient(options?: CacheServiceOptions | undefined): CacheService;
           };
         }
-      >
+      > & {
+        tokenManager: ServerTokenManager;
```

Comment:
Skipping the breakout of a LegacyTokenManager for now?

Reply:
Got that coming in the full removal of token manager, although this should be using the `TokenManager` interface instead.